readme.md

```diff
@@ -39,16 +39,26 @@ exampleConfig
   .loadEnvironment({
     prefix: 'MYAPP_ALT_NAME_', // Default is MYAPP_
     pathSeparator: '--', // Default is __
-    wordSeparator: '-',  // Default is _
-    casing: 'snake',     // Default is camel
+    wordSeparator: '-', // Default is _
+    casing: 'snake', // Default is camel
     encodeDataUris: false, // Default is true
   })
   .loadCliArguments()
   .loadDefaultFiles()
-  .loadFile({absolutePath: '/path/to/config-file.yaml'})
+  .loadFile({
+    absolutePath: '/path/to/config-file.yaml', // Supports .yaml, .json, .js
+    isRequired: true // Default is false
+    shallPrintWarning: false, // Default is true
```

Comment:
Better be default false. A library should be silent.

Reply:
If you specify a file to load, but it's actually a directory, then it should definitely throw an error.